src/utils/randomIdUtils.ts

```diff
@@ -0,0 +1,14 @@
+declare global {
+    interface Window {
+        msCrypto: {};
+    }
+}
+
+export const randomCallId = () => {
```

Comment:
Denne er tatt fra en annen kodebase i NAV.
Genererer en tilfeldig ID for å kunne enklere spore kallet til aktørregisteret i Kibana

Reply:
Går også an å bruke et lib om ønsket: https://www.npmjs.com/package/uuidv4